### PR TITLE
Lab 2

### PR DESCRIPTION
**Pull request by Maxkoz777**

### CODE REVIEW

File: app_python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.9
+
+WORKDIR /app_python
+
+COPY requirements.txt requirements.txt
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
```
Comment (AKurmazov) [inline app_python/Dockerfile:9]: nit: Avoid copying unnecessary files

**Comment by AKurmazov** [comment]
LGTM 👍

File: app_java/pom.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <version>2.7.3</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>com.example</groupId>
+  <artifactId>app_java</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>app_java</name>
+  <description>app_java</description>
+  <properties>
+    <java.version>17</java.version>
```
Comment (afonyaa) [inline app_java/pom.xml:17]: Try java 18, there are a lot of new features

**Comment by afonyaa** [comment]
Good job Maxim!

File: app_java/pom.xml (hunk 2 above)
Comment (Maxkoz777) [reply app_java/pom.xml:17]: Java 17 is the latest stable version, none of the features introduced in 18th version will somehow improve my project + Java 20 is out, so you could offer me that version)

**Comment by none-word** [comment]
Well done!

File: app_java/pom.xml (hunk 2 above)
Comment (afonyaa) [reply app_java/pom.xml:17]: Maksim, Try java 20

**Comment by Homa3030** [comment]
:+1:

**Pull request closed without merging**